docs/ide-integrations/visual-studio/Visual-Studio-Integration-Editing-Features.md

```diff
@@ -0,0 +1,137 @@
+
+# Editing Features
+
+The Visual Studio integration includes the following features to make it
+easier to edit feature files and identify which steps have already been
+bound.
+
+```{note}
+Many of the formatting behaviors can be controlled by an [.editorconfig file](editorconfig.md) and the `ide` section of the [`reqnroll.json` config file](reqnroll_json_ide_section_description.md).
+```
+
+## Gherkin Syntax Highlighting
+
+Various default styles have been defined for the Gherkin syntax. You can
+customize these colours in Visual Studio\'s settings (**Tools \| Options
+\| Environment \| Fonts and Colors**). The names of the corresponding
+**Display items** in the list begin with \"Reqnroll\".
+
+In addition to highlighting keywords, comments, tags etc., unbound steps
+and parameters in feature files are highlighted when editing the file in
+Visual Studio. The following syntax highlighting is used by default:
+
+-   Purple: unbound steps
+-   Black: bound steps
+-   Red: parameters in bound steps
+
+You can thus tell immediately which steps in a feature file have been
+bound.
+
+```{note}
+In Reqnroll Visual Studio 2022 plugin a project must be **built** for
+syntax highlighting to update.
+```
+
+## IntelliSense (auto-completion) for Keywords and Steps
+
+IntelliSense makes Reqnroll easy to use when integrated with Visual
+Studio. IntelliSense uses find-as-you-type to restrict the list of
+suggested entries.
+
+### Gherkin Files
+
+IntelliSense is available in feature files for the following:
+
+-   Gherkin keywords (e.g. `Scenario`, `Given` etc.)
+-   Existing steps are listed after a `Given`, `When` or `Then`
+    statement, providing quick access to your current steps definitions.
+
+```{note}
+Note that all the steps in all \"\*.feature\" files\" that match the
+current type (Given, When, Then) are displayed
+```
+
+![image](../../_static/images/vs2022inteli.png)
+
+
+### Code Files
+
+IntelliSense is also available for the Gherkin keywords in your code
+files.
+
+### IntelliSense settings
+
+You must edit the [reqnroll.json](../../installation/configuration.md) config file access other settings as per below:
+
+![IntelliSense 2022](../../_static/images/inteli2022.png)
```

Comment:
the filenames of this image `inteli2022.png` and `inteli2022.png` used in the previous section are misleading. It took me some time to spot the difference between `vs2022inteli` and `inteli2022.png` :)